### ISSUE TITLE
Configuration: support `markdownlint-cli2`-formatted configuration

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#547]
### Problem or motivation

`rumdl` conveniently provides compatibility with `markdownlint` configuration files: https://github.com/rvben/rumdl/blob/9ce5d163deb00a519597e730f3981c33219752d1/docs/markdownlint-comparison.md#4-configuration-philosophy.

It would be useful if `rumdl` also provided compatibility with `markdownlint-cli2` configuration files: https://github.com/DavidAnson/markdownlint-cli2/blob/3766ad839ad3a74558031510b3bc9872bb1d9980/README.md#configuration, which are slightly different. At the very least, the `config` property would allow `rumdl` to pick up rules configuration.

### Proposed solution

It might be possible for either of these styles of configuration to work:

- `markdownlint`-formatted configuration. For example:
  ```yaml
  ---
  MD049:
    style: asterisk
   ```
- `markdownlint-cli2`-formatted configuration. For example:
  ```yaml
  ---
  config:
    MD049:
      style: asterisk
  ```

Then on a file such as:

```markdown
---
title: Heading
---

Some _bold text_.
```

Both of these invocations of `rumdl` would catch the problem:

- `rumdl check --config .markdownlint.yaml` (at the moment, this catches the problem)
- `rumdl check --config .markdownlint-cli2.yaml` (at the moment, this doesn't catch the problem)

### Alternatives considered

None.

### Feature category

CLI / configuration

**Comment by rvben** [comment]
Implemented in v0.1.62. rumdl now supports [markdownlint-cli2 configuration format](https://rumdl.dev/comparison) — the `config:` wrapper key is automatically detected and unwrapped. Auto-discovery of `.markdownlint-cli2.jsonc`, `.markdownlint-cli2.yaml`, and `.markdownlint-cli2.yml` files is also included. Thanks for the suggestion, @eread!
